firmware/brake/kia_soul_ev_niro/src/brake_control.cpp

```diff
@@ -200,7 +211,27 @@ static void read_brake_pedal_position_sensor(
     brake_pedal_position_s * const value )
 {
     cli();
-    value->high = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_HIGH );
-    value->low = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_LOW );
+    value->A = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_A );
+    value->B = analogRead( PIN_BRAKE_PEDAL_POSITION_SENSOR_B );
     sei();
 }
+
+uint8_t check_brake_pedal_position_data(
+    brake_pedal_position_s * const value )
+{
+    uint8_t error_count = 0;
+    if( value->A > (BRAKE_SPOOF_A_SIGNAL_RANGE_MAX >> 2))
```

Comment:
What's the meaning of these right shifts? Is this to deal with the ADCs having 10 bit resolution?